### PR TITLE
add css to admin pages. all tests are passing

### PR DESCRIPTION
**Pull request by mbburch**
Hi guys. I added CSS to cart and admin pages. It's not perfect by any means... still bootstrappy, but I think it looks better. You can decide and see what you think. It can't automatically merge. Not sure where the conflicts are til I do the pull request. My guess is gemfile crap again?? All tests are passing.

### CODE REVIEW

File: app/assets/stylesheets/shop-homepage.scss [hunk 1]
```diff
@@ -60,3 +60,7 @@ $grey: #333;
   height: auto;
   width: 200px;
 }
+
+#checkout a {
```
Comment (mbburch) [inline app/assets/stylesheets/shop-homepage.scss]: Kept this as I was intentionally changing text color in one place.

File: app/assets/stylesheets/custom.scss [hunk 2]
```diff
@@ -13,3 +13,11 @@ body {
 .slide-image {
   width: 100%;
 }
+
+#admin-links {
+  padding-bottom: 15px;
+}
+
+#admin-links a {
```
Comment (mbburch) [inline app/assets/stylesheets/custom.scss]: I had to specify a to get link to reflect change.

**Comment by mbburch** [comment]
OK, the merge conflict was the gemfile. All tests still pass. I reloaded a ton and clicked the navbar and admin buttons with no issues. Fingers crossed.

**Comment by mbburch** [comment]
OK, going to bed. Let's be sure to go over this with wireframes tomorrow.

**Comment by russelleh** [comment]
Bootstrappin' ;-)

**Comment by jbrr** [comment]
I'm not sure that we should remove turbolinks. It was working yesterday with turbolinks, and the whole thing seems so fragile that I don't really want to mess with it. Otherwise, cool! I think we should look at this, because the CSS is kinda hard to read through just as a git diff.

**Comment by jbrr** [comment]
I'm digging the orders page!

**Comment by jbrr** [comment]
Looks rad, we should merge it

**Pull request merged**